static/scss/ecommerce.scss

```diff
@@ -0,0 +1,70 @@
+.payment-drawer {
+  max-width: 600px;
+  padding: 25px;
+
+  .drawer-title {
```

Comment:
Can we make an effort to reuse existing classes/definitions, and if they don't exist, create new ones that are likely to be reused? Examples:
- The `h2` definition in `_shared.scss` is very close to this `.drawer-title` definition. This has wider letter spacing, but I'm willing to bet it won't matter if we use that already-defined letter spacing.
- `.bootcamp` and `.payment-amount` have the same `font-size` and `letter-spacing` .
- `button` probably has some styles that we would want to share with other buttons. Maybe the designs show a slight discrepancy with this button and other buttons, but I think our approach should be to use the same styling if any of these elements look moderately similar. If the discrepancies really were intentional, Abdul can let us know before we go live. For now I think it's much more likely that we're going to have barely-controlled expansion of css without a real need for it

Reply:
I made some changes. Let me know if you spot other places we can consolidate CSS. I'm not really sure what the threshold is for new CSS vs reusing slightly different CSS. 
![Screenshot from 2020-06-04 16-10-57](https://user-images.githubusercontent.com/863262/83805721-65122700-a67e-11ea-9130-0424cd43c6ca.png)